kogito-kafka-quickstart-springboot/README.md

```diff
@@ -59,28 +118,66 @@ java -jar target/kogito-kafka-quickstart-sprintboot-{version}.jar
 
 To make use of this application it is as simple as putting a message on `travellers` topic with following content 
 
+* To examine ProcessedTravellers topic and verify upcoming messages will be processed
```

Comment:
I would add to execute this in a different terminal

Reply:
fixed @radtriste